main.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"cloud.google.com/go/storage"
+	"context"
+	"flag"
+	"fmt"
+	"google.golang.org/api/iterator"
+	"google.golang.org/api/option"
+	"gopkg.in/yaml.v2"
+	"log"
+	"os"
+	"time"
```

Comment:
try to create different sections for built-in library vs user-defined ones,
```suggestion
	"context"
	"flag"
	"fmt"
	"log"
	"os"
	"time"
	
	"cloud.google.com/go/storage"
	"google.golang.org/api/iterator"
	"google.golang.org/api/option"
	"gopkg.in/yaml.v2"
```